### PR TITLE
ss-263:  fixed styles in user profile setting page

### PR DESCRIPTION
**Pull request by anton-otroshchenko**
image

### CODE REVIEW

File: frontend/src/bundles/account-settings/components/settings/user-profile/settings-form.tsx [hunk 1]
```diff
@@ -136,6 +143,17 @@ const SettingsForm: React.FC = () => {
             <SubmitButton isChange={isChange()}>
                 Update My Settings
             </SubmitButton>
+            <div className={styles.dltButton}>
+                <Button variant={ButtonVariant.DELETE} size={ButtonSize.MEDIUM}>
+                    <FontAwesomeIcon
```
Comment (VItaliy-Kalenichenko) [inline frontend/src/bundles/account-settings/components/settings/user-profile/settings-form.tsx:148]: Don't use FontAwesomeIcon directly. Move it to a separate component. 
You can discuss the creation of this component with @ValeriiArt, as he has the same comments

File: frontend/src/bundles/account-settings/components/settings/user-profile/settings-form.tsx [hunk 2]
```diff
@@ -136,6 +143,17 @@ const SettingsForm: React.FC = () => {
...
+                        style={{
+                            paddingRight: '5px',
+                        }}
```
Comment (VItaliy-Kalenichenko) [inline frontend/src/bundles/account-settings/components/settings/user-profile/settings-form.tsx:151]: Try to find a way to use className

Comment (anton-otroshchenko) [reply frontend/src/bundles/account-settings/components/settings/user-profile/settings-form.tsx]: fixed

File: frontend/src/bundles/account-settings/components/settings/user-profile/settings-form.tsx (hunk 1 above)
Comment (anton-otroshchenko) [reply frontend/src/bundles/account-settings/components/settings/user-profile/settings-form.tsx]: fixed

File: frontend/src/bundles/account-settings/components/settings/styles.module.scss [hunk 3]
```diff
@@ -54,10 +54,10 @@
     width: 100%;
     height: 100%;
     padding: var(--p-2);
-    color: var(--color-green-300);
+    color: #01cacc;
```
Comment (nikita-remeslov) [inline frontend/src/bundles/account-settings/components/settings/styles.module.scss:57]: mb we should use variable for it

**Review by nikita-remeslov — APPROVED** [review]

**Pull request merged**